src/trading-api/draft/perps/bullish-trading-api.yml

```diff
@@ -3998,6 +3999,10 @@ components:
           description: whether the trading account is defaulted
           type: String
           example: "false"
+        riskLimitUSD:
+          description: taker fee
```

Comment:
- is this description correct?

Reply:
updated description: The maximum allowed borrowing for this trading account (in USD currency)